docs/attributes-registry/cicd.md

```diff
@@ -0,0 +1,28 @@
+<!--- Hugo front matter used to generate the website version of this page:
+--->
+
+<!-- NOTE: THIS FILE IS AUTOGENERATED. DO NOT EDIT BY HAND. -->
+<!-- see templates/registry/markdown/attribute_namespace.md.j2 -->
+
+# CICD
```

Comment:
nit: I think you can now specify display_name in yamls and customize how it shows up

Reply:
Looks like `display_name` affects the `H2` value instead of the root only. I'm fine with `CICD` being the `H1` value. Going to resolve this, but feel free to reopen!

<img width="874" alt="image" src="https://github.com/user-attachments/assets/c9fbb25b-761c-4b16-b5b3-c861667a4307">